bridgerelayer/magefile.go

```diff
@@ -0,0 +1,15 @@
+//+build mage
+
+package main
+
+import (
+	"github.com/magefile/mage/sh"
```

Comment:
Wondering why we've switched to Magefile instead of Makefile?

Reply:
yeah I'm not a fan of using Makefiles as a means of running arbitrary tasks. Along the lines of what the Mage author said, they can end up being a confusing tangle of Bash scriptlets with interspersed with Make syntax. At least that was my experience on previous projects.

Also, Make isn't trivially installable on Windows, which reduces cross-platform compatibility.